client/game/board/Board.svelte

```diff
@@ -2,29 +2,44 @@
 	export let cards = [];
 
 	import Card from "./Card.svelte";
-	import { createEventDispatcher } from "svelte";
+	import { socket } from "../../connectivity.js";
 	import { areCardsEqual, arrayContainsCard, isValidPlay } from "../shared.js";
 
-	const dispatch = createEventDispatcher();
+	const keyboardMap = ["qwertyu", "asdfghj", "zxcvbnm"].map(line => line.split(""));
+
+	function getKey(i) {
+		if (i >= keyboardMap.length * keyboardMap[0].length) {
+			return "";
+		} else {
+			return keyboardMap[i % 3][parseInt(i / 3)].toUpperCase();
+		}
+	}
 
 	let selection = [];
 
 	function cardClicked(e) {
+		if (selection.length >= 3) {
+			return;
+		}
 		const card = e.detail.card;
 		if (arrayContainsCard(selection, card)) {
 			selection = selection.filter(card => !areCardsEqual(card, e.detail.card));
 		} else {
 			selection = [...selection, card];
 			if (selection.length == 3) {
```

Comment:
```suggestion
			if (selection.length === 3) {
```

Reply:
Since the length of an array will always be an integer, I think it will never come to type ambiguities here. In the case of `undefined`, it will never be equal to 3, no?